lib/main.js

```diff
@@ -15,104 +15,133 @@ let Matrix = require('./models/matrix');
 import WAAClock from 'waaclock';
 import Interval from './time/interval';
 
+const DefaultContext = window.AudioContext || window.webkitAudioContext;
+
+function unlockAudioContext(audioContext) {
+  const events = ['touchstart', 'touchend', 'mousedown', 'keydown'];
+
+  function unlock() {
+    events.forEach((eventName) => document.body.removeEventListener(eventName, unlock));
+    if (audioContext.state === 'suspended') {
+      console.log('Audio context - unlocked ...');
+      audioContext.resume();
+    }
+  }
+
+  if (audioContext.state === 'suspended') {
+    console.log('Audio context - suspended until user interaction ...');
+    events.forEach(eventName => {
+      document.body.addEventListener(eventName, unlock, false);
+    });
+  }
+}
```

Comment:
Not sure about this. Similar code seems to be used by multiple WebAudio projects (because ```AudioContext``` is supposed to initialize in a "suspended" state). Recent WebAudio issues indicate that this is implemented differently between the different browsers.